av_camera_launch/launch/all_cams.launch.xml

```diff
@@ -1,14 +1,20 @@
 <launch>
   <group>
-    <push-ros-namespace namespace="sensor/camera"/>
+    <push-ros-namespace namespace="/sensor/camera"/>
     <node_container pkg="rclcpp_components" exec="component_container_mt" name="camera_container" namespace="">
-      <!-- fsp_l -->
-
-      <composable_node pkg="spinnaker_camera_driver" plugin="spinnaker_camera_driver::CameraDriver" name="fsp_l" namespace="">
+      <!-- Synchronised Camera Driver composable node -->
+      <composable_node pkg="spinnaker_synchronized_camera_driver" plugin="spinnaker_synchronized_camera_driver::SynchronizedCameraDriver" name="camera" namespace="/sensor">
```

Comment:
Wouldn't the composable node grasp the `namespace` defined above? 

Can we call this node `spinnaker_camera_sync_node` ? :)